packages/nimble/R/genCpp_exprClass.R

```diff
@@ -35,7 +34,6 @@ exprClass <- setRefClass('exprClass',
             toEigenize <<- 'unknown'
             assertions <<- list()
             eigMatrix <<- logical()
-            cppADCode <<- FALSE
```

Comment:
Why don't you set the default value any more?

Reply:
Oh, is this because you've moved the state to a global variable? (if so, I'll have to fix my tensorflow code)

Reply:
@fritzo That's correct.  A suggestion @perrydv had that I agreed with was to set a global variable instead of a per-expression variable, as the latter seems somewhat more involved than may be necessary.

Reply:
I wish this thread could be marked 'resolved'. Vote for that feature here: https://github.com/dear-github/dear-github/issues/154